.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,27 @@
+name: GitHub Actions Demo
```

Comment:
I think that this GitHub Actions Demo is not needed.

Reply:
Ok